### ISSUE TITLE
[fonts] Outline Circle Icon Bug

### ISSUE THREAD

**Issue by zackkrida** [cc-archive/vocabulary-legacy#765]
## Screenshot

<img width="191" alt="Screen Shot 2020-11-04 at 3 47 32 PM" src="https://user-images.githubusercontent.com/6351754/98190453-24336e00-1ebb-11eb-93d6-9fb5c4754dd7.png">

The `circle-outline` icon has a black fill. Adding `fill: transparent` seems to fix but there might be a better solution.

**Comment by obulat** [comment]
To optimize the size of the icons, I used both svgomg and manual changes. With this icon, I took the liberty and replaced an `<svg>` path that I got from Figma with a `<circle>` svg element. 

Figma adds a `fill="none"` attribute to icons' `<svg>` elements when exporting them. (All icons from 'heroicons' icon set also have the fill set to none, by the way). I removed all of those attributes from the Figma exports and didn't see any changes, so I thought they were redundant. 

So, I think the best way to fix this problem is to simply add fill="none" to the top svg element.  Or I could also go back to path (and maybe still add the fill property), if there is some potential issue I'm not aware of.
